### PR TITLE
Added Czech and Swedish translations

### PR DESCRIPTION
**Pull request by xmstspider**
Hi,

I translated your extension to Czech and Swedish.

### CODE REVIEW

**Comment by Molkobain** [comment]
Hello @xmstspider thank you so much for your contribution! 🙌

I'll take a look this WE and integrate it.
Could you tell me if you would like to appear on the contributors list by your fullname or only your nickname? (If fullname, please provide it)

Cheers,
Guillaume

**Comment by xmstspider** [comment]
Hello, happy to help :-)

You can use my full name: Zdenek Sedlak.

Thanks for the good work :-)

File: dist/molkobain-datacenter-view/molkobain-datacenter-view/cs.dict.molkobain-datacenter-view.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+	// Fieldsets
+	'Fieldset:baseinfo' => 'Obecné informace',
+	'Fieldset:moreinfo' => 'Více informací',
+	'Fieldset:otherinfo' => 'Další informace',
+	'Fieldset:dates' => 'Datumy',
+]);
+
+// Classes
+Dict::Add('CZ CZ', 'Czech', 'Česky', array(
```
Comment (Molkobain) [inline dist/molkobain-datacenter-view/molkobain-datacenter-view/cs.dict.molkobain-datacenter-view.php:28]:
```suggestion
Dict::Add('CS CZ', 'Czech', 'Čeština', array(
```

In iTop existing czech dictionnaries are using another code / original language name like the suggestion I put.
I don't know which one is the correct one, but we need to keep it consistent with the other files for it to work well in all circumstances.

File: dist/molkobain-datacenter-view/molkobain-datacenter-view/cs.dict.molkobain-datacenter-view.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+	'Class:DatacenterDevice/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:device'], 'zařízení'),
+	// - PDU
+	'Class:PDU/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:PDU/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'zařízení'),
+	'Class:PDU/Attribute:position_v' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v']),
+	'Class:PDU/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:device'], 'napájení'),
+));
+
+// UI
+Dict::Add('CZ CZ', 'Czech', 'Česky', array(
```
Comment (Molkobain) [inline dist/molkobain-datacenter-view/molkobain-datacenter-view/cs.dict.molkobain-datacenter-view.php:73]:
```suggestion
Dict::Add('CS CZ', 'Czech', 'Čeština', array(
```

Same as the previous one.

File: dist/molkobain-datacenter-view/molkobain-datacenter-view/sv.dict.molkobain-datacenter-view.php [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+	'Class:DatacenterDevice/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:device'], 'enhet'),
+	// - PDU
+	'Class:PDU/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:PDU/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'enhet'),
+	'Class:PDU/Attribute:position_v' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v']),
+	'Class:PDU/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:device'], 'Strömenhet'),
+));
+
+// UI
+Dict::Add('EN US', 'English', 'English', array(
```
Comment (Molkobain) [inline dist/molkobain-datacenter-view/molkobain-datacenter-view/sv.dict.molkobain-datacenter-view.php:73]:
```suggestion
Dict::Add('SV SV', 'Swedish', 'Svenska', array(
```

Typo, the old "english" header was kept.

File: src/molkobain-datacenter-view/cs.dict.molkobain-datacenter-view.php [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+	// Fieldsets
+	'Fieldset:baseinfo' => 'Obecné informace',
+	'Fieldset:moreinfo' => 'Více informací',
+	'Fieldset:otherinfo' => 'Další informace',
+	'Fieldset:dates' => 'Datumy',
+]);
+
+// Classes
+Dict::Add('CZ CZ', 'Czech', 'Česky', array(
```
Comment (Molkobain) [inline src/molkobain-datacenter-view/cs.dict.molkobain-datacenter-view.php:28]:
```suggestion
Dict::Add('CS CZ', 'Czech', 'Čeština', array(
```

Same as for the previous czech file.

File: src/molkobain-datacenter-view/cs.dict.molkobain-datacenter-view.php [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+	'Class:DatacenterDevice/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:device'], 'zařízení'),
+	// - PDU
+	'Class:PDU/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:PDU/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'zařízení'),
+	'Class:PDU/Attribute:position_v' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v']),
+	'Class:PDU/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:device'], 'napájení'),
+));
+
+// UI
+Dict::Add('CZ CZ', 'Czech', 'Česky', array(
```
Comment (Molkobain) [inline src/molkobain-datacenter-view/cs.dict.molkobain-datacenter-view.php:73]:
```suggestion
Dict::Add('CS CZ', 'Czech', 'Čeština', array(
```

Same as for the previous czech file.

File: src/molkobain-datacenter-view/sv.dict.molkobain-datacenter-view.php [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+	'Class:DatacenterDevice/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:device'], 'enhet'),
+	// - PDU
+	'Class:PDU/Attribute:nb_u' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u']),
+	'Class:PDU/Attribute:nb_u+' => sprintf($aMolkobainThesaurus['Class/Attribute:nb_u+'], 'enhet'),
+	'Class:PDU/Attribute:position_v' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v']),
+	'Class:PDU/Attribute:position_v+' => sprintf($aMolkobainThesaurus['Class/Attribute:position_v+/Variant:device'], 'Strömenhet'),
+));
+
+// UI
+Dict::Add('EN US', 'English', 'English', array(
```
Comment (Molkobain) [inline src/molkobain-datacenter-view/sv.dict.molkobain-datacenter-view.php:73]:
```suggestion
Dict::Add('SV SV', 'Swedish', 'Svenska', array(
```

**Review by Molkobain — CHANGES_REQUESTED** [review]
A few changes are needed on some dictionaries headers before integration please.

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
Thanks for the update !

**Pull request merged**